conans/model/conan_file.py

```diff
@@ -48,6 +48,7 @@ class ConanFile:
     default_options = None
     default_build_options = None
     package_type = None
+    languages = []
```

Comment:
Can't forget to serialize this, and make it available in the conanfile interface